app/scripts/controllers/citations.ctrl.js

```diff
@@ -17,12 +17,12 @@ angular.module('yourStlCourts').controller('CitationsCtrl', function (Citations,
           $state.go('citationInfo', {citations: result  });
         } else {
           var homeLink = '<a href="/"><u>clicking here</u></a>';
-          var noTicketsFoundMsg = 'We could not find any results for the  information you provided. It\'s possible that the municipality that issued your citation does not participate in YourSTLCourts. You may obtain information for any municipality via '+homeLink+'. Mention you\'d like them to participate in YourSTLCourts.';
+          var noTicketsFoundMsg = 'We could not find any results for the information you provided. Because only recent court dates are in our system, you may still have pending court cases, please contact the court for more information.<br>It\'s also possible that the municipality that issued your citation does not participate in YourSTLCourts. You may obtain information for any municipality via '+homeLink+'. Mention you\'d like them to participate in YourSTLCourts.';
```

Comment:
@wstampley Should this messages file be a json file or something on the server?  (so that it is easier to change via the admin file?